panedr/panedr.py

```diff
@@ -421,11 +421,28 @@ def edr_to_df(path, verbose=False):
             all_energies.append([frame.t] + [ener.e for ener in frame.ener])
 
     end = time.time()
-    if verbose:
+    if verbose_set:
         print('\rLast Frame read : {}, time : {} ps'
               .format(ifr, frame.t),
               end='', file=sys.stderr)
         print('\n{} frame read in {:.2f} seconds'.format(ifr, end - begin),
               file=sys.stderr)
+
+    return all_energies, all_names, times
+
+
+def edr_to_df(path: str, verbose: bool = False):
+    import pandas
```

Comment:
Is pandas now an optional dependency in requirements.txt etc? If so I would guard this with a `try: except` similar to 

```python
        try:
            import pandas
        except ImportError:
            raise ImportError("""ERROR --- pandas was not found!
                pandas is required to use the `.edr_to_df()` functionality.
                try installing it using pip eg:
                    pip install pandas """)

```

Reply:
Good point. I'll make a note to add a test for raising this error as well.

Reply:
I wasn't sure what the best way to make pandas optional is. I have now done this by removing pandas from requirements.txt and adding a section under [extras] in setup.cfg. panedr can now be installed with pandas by running 

`pip install -e .[pandas]`

Please let me know if this is not the best way to do these things.

Reply:
I would rather do things a little bit differently. The way you do it here will break the user experience for the standalone case. A user who do not use mdanalysis will install panedr and won't have pandas to use the main function out of the box.

Instead, I would create 2 packages: the default one that depends on pandas and a "lite" one for downstream integrators who want to minimise dependencies.

I don't know how to do that, though...